lib/syskit/cli/gen/syskit_app/config/robots/robot.rb.erb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
```

Comment:
Robot.config is replaced by Robot.setup. `config` was a confusing name from Syskit's perspective. Check out the comment.

If you are confused by this, maybe we should find a way to make it more obvious. Do you have a suggestion ?

Reply:
I dont have a problem with the name. It's more that it wasnt clear what were all the possible steps, and since some of the code I've seen used some of the old names (i.e. Robot.config) I wasnt sure what was the exact order of each stage. Maybe add a deprecated warning on the replaced names?